ansible/roles/prosody/templates/prosody-jvb.cfg.lua.j2

```diff
@@ -45,8 +45,8 @@ firewall_scripts = {
 
 modules_disabled = {
     "offline",
-    "component",
```

Comment:
"component" is not loaded be default?

Reply:
Nope. It is loaded when you define a component. If you try to load it as a module you get this error:
https://github.com/bjc/prosody/blob/98795d0b6eb9ed15063e8699b269abc30271d106/plugins/mod_component.lua#L44